test/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -47,11 +47,14 @@ spec:
             - containerPort: {{ .Values.controller.metricsPort }}
               name: metrics
               protocol: TCP
+            - containerPort: 10000
```

Comment:
This should be `{{ .Values.controller.port }}`

Reply:
Fixed